test/device/test_device_configuration.py

```diff
@@ -446,11 +446,13 @@ def test_cannot_load_config_from_when_path_or_from_env_var_not_provide(moler_con
     assert "Provide either 'config' or 'from_env_var' parameter (none given)" in str(err.value)
 
 
-def test_cannot_load_config_from_when_wrong_config_type_provided(moler_config):
+@pytest.mark.parametrize('params', [{'config': (), 'config_type': "wrong_type"},
+                                    {'config': ()}])
+def test_cannot_load_config_from_when_wrong_config_type_provided(moler_config, params):
     with pytest.raises(WrongUsage) as err:
-        moler_config.load_config(config_type="wrong_type")
+        moler_config.load_config(**params)
 
-    assert "Unsupported config_type" in str(err.value)
+    assert "Unsupported config type" in str(err.value)
 
 
```

Comment:
missing test for case where both params provided and config has wrong type:
` from_env_var="AAA" and config=[1,2,3]`

Reply:
0fef3d93c7b7e81dfec3d4ae2e826dbe5d57d139